src/model/version.entity.ts

```diff
@@ -0,0 +1,23 @@
+import { Entity, PrimaryKey, Property, ManyToOne } from '@mikro-orm/core';
+import { Package } from './package.entity';
+
+@Entity()
+export class Version {
+  @PrimaryKey()
+  id!: number;
+
+  @ManyToOne(() => Package)
+  package!: Package;
+
+  @Property()
+  version!: string;
+
+  @Property({ columnType: 'text' })
+  packageBlob!: string;
```

Comment:
Maybe:
  ```
  @Property({ type: 'blob' }) // Maps to BLOB in the database
  data!: Buffer;
```

then we will use blob data. Here we store .tar.gz. It should be also easy to download file